### PR TITLE
class setup

### PR DESCRIPTION
**Pull request by TziKwan**

### CODE REVIEW

**Comment by TziKwan** [comment]
hey man, looks good to me.

**Comment by AymenQ** [comment]
Looks good 👍

File: src/Mancala.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+public class Mancala extends Pocket{
```
Comment (AymenQ) [inline src/Mancala.java:1]: Excellent use of inheritance.

File: src/MancalaBoard.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+public class MancalaBoard {
+
+
+}
```
Comment (rjhayward) [inline src/MancalaBoard.java:4]: An excellent closing curly bracket

**Pull request merged**